core/data/rules.yml

```diff
@@ -186,24 +223,40 @@ rules:
       - action: utter_ask_anything_else_without_test_navigation
 
   ## Return for check-in rules
-  - rule: Check-in return severe symptoms
-    condition:
-      - active_loop: checkin_return_form
-    steps:
-      - action: checkin_return_form
-      - active_loop: null
-      - slot_was_set:
-          - symptoms: severe
-      - action: action_severe_symptoms_recommendations
-
   - rule: Check-in return moderate symptoms
     condition:
-      - active_loop: checkin_return_form
+      - active_loop: assessment_form
+      - slot_was_set:
+          - assessment_type: checkin_return
     steps:
-      - action: checkin_return_form
+      - action: assessment_form
       - active_loop: null
       - slot_was_set:
           - symptoms: moderate
+      - action: utter_ask_checkin_return__moderate_symptoms_worsened
+
+  - rule: Check-in return moderate symptoms worsened
+    steps:
+      - action: utter_ask_checkin_return__moderate_symptoms_worsened
+      - intent: affirm
+      - action: utter_contact_healthcare_professional
+      - action: utter_contact_healthcare_professional_options
+      - action: utter_symptoms_worsen_emergency
+      - action: utter_returning_self_isolate
+      - action: utter_self_isolation_final
```

Comment:
I know this is a lot of messages but I wasn`t sure on how to regroup them so I left them like this for now.